Update README.md

hirehamir opened the pull request:
Adding https://github.com/tiborpilz/virtuaw-web as requested in #63.

hirehamir committed:
Update README.md

Adding https://github.com/tiborpilz/virtuaw-web as requested in #63.

amilajack reviewed (changes requested):
Lets move this to a new daw section

hirehamir committed:
Update README.md

Created a new Digital Audio Workstation (DAW) section for [VirtuAW - The Virtual Audio Workstation](https://github.com/tiborpilz/virtuaw-web), as requested in #63.

hirehamir reviewed (commented):
Created a new section for Digital Audio Workstations (DAWs) as requested and moved the link from #63 there.

amilajack reviewed (approved):
nice! 🎉
